### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #902 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#434]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 1.68 KB...]
Downloading: https://jitpack.io/net/minidev/json-smart/maven-metadata.xml
[INFO] 
[INFO] --- maven-clean-plugin:2.6.1:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 1 resource
[INFO] 
[INFO] --- maven-compiler-plugin:3.6.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 75 source files to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/AddReviewService.java:[20,51] cannot find symbol
  symbol:   class AddReviewFailed
  location: package smartcity.accessibility.services.exceptions
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/AddReviewService.java:[62,51] cannot find symbol
  symbol:   class AddReviewFailed
  location: class smartcity.accessibility.services.AddReviewService
[INFO] 2 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 14.935 s
[INFO] Finished at: 2017-06-14T19:53:54+00:00
[INFO] Final Memory: 32M/213M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/AddReviewService.java:[20,51] cannot find symbol
[ERROR] symbol:   class AddReviewFailed
[ERROR] location: package smartcity.accessibility.services.exceptions
[ERROR] /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/src/main/java/smartcity/accessibility/services/AddReviewService.java:[62,51] cannot find symbol
[ERROR] symbol:   class AddReviewFailed
[ERROR] location: class smartcity.accessibility.services.AddReviewService
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] 82a674eefc74b6bcbe8b9dc435befc89daa7abee - add segment id to reviews on street #433

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/902/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
